### PR TITLE
[DEV APPROVED]Allow editor to create pages

### PR DESCRIPTION
**Pull request by jriga**
**Feature**
[card 7701](https://moneyadviceservice.tpondemand.com/entity/7701)
As a Welsh Translator, I need to be able to create and edit draft pages in Contento, so that I can create Welsh versions of pages and thus lighten the workload of the MAS internal content team.

**Proposed Solution**
 - [x] Amend the privileges of 'Editor' users to enable them to create new draft pages.

### CODE REVIEW

**Comment by Tr4pSt3R** [comment]
👍

File: spec/lib/permission_check_spec.rb [hunk 1]
```diff
@@ -49,15 +49,15 @@
...
+          expect(subject).to be true
         end
       end
 
       context 'and the action is create' do
         let(:action) { 'create' }
 
         it 'returns false' do
-          expect(subject).to be false
+          expect(subject).to be true
```
Comment (rnalexander) [inline spec/lib/permission_check_spec.rb]: Same comment as above, need to change the description text when you change the test expectations.

Comment (jriga) [inline spec/lib/permission_check_spec.rb]: ♻️

Comment (jriga) [inline spec/lib/permission_check_spec.rb]: This test says that editors can now create the new draft article

**Comment by tomas-stefano** [comment]
👍

**Comment by rnalexander** [comment]
:+1: :shipit:

**Pull request merged**